### PR TITLE
added appeal

### PR DESCRIPTION
**Pull request by WilloIzCitron**
i added appeal command for appealing on discord, mindustry, terraria and minecraft.

### CODE REVIEW

**Comment by alexpvpmindustry** [comment]
Ahh nice, but without a compiler I can already see 2 syntax errors. Please use an ide.

**Comment by WilloIzCitron** [comment]
i using with phone

**Comment by alexpvpmindustry** [comment]
That's no excuse to make unrunnable code. Also, there's probably ide mobile app. (not that I recommend programming on phone)

**Comment by WilloIzCitron** [comment]
ok i fixed
with repl.it

File: mastermelon/melon.py [hunk 1]
```diff
@@ -455,6 +455,24 @@ async def convertexp(ctx: discord.ext.commands.Context):
...
+@bot.command
+async def appeal(ctx, args, punishment: str, idoruuid: str = None, reason: str = None):
+     if isinstance(idoruuid, type(None)):
+        return ctx.channel.send("you must fill a player id or uuid")
+     if not punishment.startwith("minecraftBan", "terrariaBan", "mindustryKick", "mindustryBan"):
+        return ctx.channel.send("you must fill a punishment type, here is:/nmindustryBan, mindustryKick, terrariaBan, minecraftBan")
+     if isinstance(reason, type(None)):
+        return ctx.channel.send("you must fill a reason of you got banned/kick")
+     else:
+          channel = discord.utils.get(ctx.guild.channels, id="")
```
Comment (WilloIzCitron) [inline mastermelon/melon.py:467]: Add a channel via channelid

Comment (WilloIzCitron) [reply mastermelon/melon.py:467]: on line 467

Comment (alexpvpmindustry) [reply mastermelon/melon.py:467]: image

Comment (alexpvpmindustry) [reply mastermelon/melon.py:467]: image
2) players dont know their uuid
1) missing id, will break the code.

**Review by alexpvpmindustry — CHANGES_REQUESTED** [review]

File: mastermelon/melon.py (hunk 1 above)
Comment (WilloIzCitron) [reply mastermelon/melon.py:467]: ok

**Comment by WilloIzCitron** [comment]
fixed

**Comment by alexpvpmindustry** [comment]
image
u know there is no such method for strings?

**Comment by WilloIzCitron** [comment]
> image
> u know there is no such method for strings?

oops i fix it

**Comment by alexpvpmindustry** [comment]
bruh. pls go to the spare server and test there... ill already gave u admin rights there. add the bot to the spare server and TEST FIRST

**Comment by WilloIzCitron** [comment]
ok im testing now

**Comment by WilloIzCitron** [comment]
image
proof of testing

**Pull request merged**